build/gulp/localization.js

```diff
@@ -15,7 +16,11 @@ const Cldr = require('cldrjs');
 const locales = require('cldr-core/availableLocales.json').availableLocales.full;
 const weekData = require('cldr-core/supplemental/weekData.json');
 const likelySubtags = require('cldr-core/supplemental/likelySubtags.json');
-const parentLocales = require('../../node_modules/cldr-core/supplemental/parentLocales.json').supplemental.parentLocales.parentLocale;
+const parentLocales = require('cldr-core/supplemental/parentLocales.json').supplemental.parentLocales.parentLocale;
+
+/* for Globalize*/
```

Comment:
Could you try to avoid such comments in the code and make the code self-documenting?